public/index.tsx

```diff
@@ -264,6 +264,7 @@ function runApp() {
       cellDownloadHref={args.cellDownloadHref}
       viewerConfig={viewerConfig}
       viewerChannelSettings={args.initialChannelSettings}
+      metadataConfig={["dimensions", "pixelPhysicalSize", "physicalDimensions", "channels", "userData"]}
```

Comment:
if none specified, what is the default behavior?

Reply:
If this prop is missing then the "Image" category won't appear in the panel.